core/utils.py

```diff
@@ -0,0 +1,24 @@
+import jwt
+
+from django.conf import settings
+from django.http import JsonResponse
+from users.models import User
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            access_token = request.headers().get('Authorization')
```

Comment:
headers는 method가 아닌 json(dict구조)이며 거기서 key가 Authorization인 것의 value를 access_token이라는 이름의 변수에 담는 코드 아닌가요? 아래 처럼 수정해도 정상 동작할 것 같습니다! 

```suggestion
            access_token = request.headers.get('Authorization')
```